integration/aggregate_stats_compat_test.go

```diff
@@ -127,18 +119,49 @@ func TestAggregateCompatCollStats(tt *testing.T) {
 						nonEmptyResults = true
 					}
 
-					// Check the returned values when possible
-					// TODO https://github.com/FerretDB/FerretDB/issues/2349
```

Comment:
Sizes of `storageStats` sanity check is done in https://github.com/FerretDB/FerretDB/pull/3538/files#diff-81f1b3fd332849d4efcb1ed0c8d183e19ecef6105d824a6a50eaf0482336ffe6